client/init.c

```diff
@@ -251,8 +251,9 @@ TDNFRefreshSack(
         /*
          * skip the @cmdline repo - options do not apply, and it is
          * initialized.
+         * In addition skip any repository that has a null repo name set
          */
-        if (!strcmp(pRepo->pszName, CMDLINE_REPO_NAME) || !pRepo->nEnabled)
+        if ((!pRepo->pszName) || !strcmp(pRepo->pszName, CMDLINE_REPO_NAME) || !pRepo->nEnabled)
```

Comment:
I think we should not just ignore and skip it. I think we should check for it in line 238, and bail if NULL.

Maybe we should also add a check right when we read the configuration.

Reply:
Can you please address this?

Reply:
@AZaugg , are going to address my comment? It's fine if not, I can take over.